source/macos/AdaptiveCards/AdaptiveCards/Views/ACRColumnView.swift

```diff
@@ -144,4 +147,19 @@ class ACRColumnView: ACRContentStackView {
             widthConstraint.isActive = false
         }
     }
+    
+    func bleedBackgroundImage(padding: CGFloat, top: Bool, bottom: Bool, leading: Bool, trailing: Bool, paddingBottom: CGFloat, with bottomAnchor: NSLayoutAnchor<NSLayoutYAxisAnchor>) {
+        backgroundImageTopConstraint.isActive = false
```

Comment:
we can activate these new constraints without turning them false, right? 
Or do they have to be turned off and then activated again?

Reply:
We have turn them off, otherwise getting conflicts of constraints error

Reply:
Okay got it